### PR TITLE
Updates of primary FCLs to match reviewers changes

### PR DESCRIPTION
**Pull request by sophiemiddleton**
As Andrei pointed out we already have a 2016 table so I deleted mine and updated it here

### CODE REVIEW

**Comment by brownd1978** [comment]
Should this be closed, now the reason for the new file is clear?

File: JobConfig/primary/CePLeadingLog.fcl [hunk 1]
```diff
@@ -10,22 +10,18 @@ physics.producers.generate : {
...
+    ehi : 92.32 
+    spectrumResolution : 0.1 #MeV
+  }
   pdgId : -11
   verbosity : 0
 }
 
 physics.producers.FindMCPrimary.PrimaryProcess : "mu2eCePlusLeadingLog"
-outputs.Output.fileName: "dts.owner.eMinusLeadingLog.version.sequencer.art"
+outputs.Output.fileName: "dts.owner.CeLeadingLog.version.sequencer.art"
```
Comment (gaponenko) [inline JobConfig/primary/CePLeadingLog.fcl:25]: The output file names for eminus and eplus clash with each other.

File: JobConfig/primary/DIOtail.fcl [hunk 2]
```diff
@@ -17,7 +17,7 @@ physics.producers.generate : {
 		        pdgId            :  11
 		        spectrumVariable : "totalEnergy"
 		        spectrumShape    : "tabulated"
-		        spectrumFileName : "ConditionsService/data/heeck_et_al_2021_aluminum.tbl"
+		        spectrumFileName : "ConditionsService/data/czarnecki_szafron_Al_2016_tail.tbl"
```
Comment (gaponenko) [inline JobConfig/primary/DIOtail.fcl:20]: I do not understand this change.   A related PR has added the "heeck" spectrum file, but now it is not used by anything?

**Review by gaponenko — CHANGES_REQUESTED** [review]
Hi Sophie,  can you please fix the file name clash and answer the "heeck" question below?
Andrei

File: JobConfig/primary/DIOtail.fcl (hunk 2 above)
Comment (sophiemiddleton) [reply JobConfig/primary/DIOtail.fcl:20]: Yes, I thought I had changed this....let me check

**Comment by kutschke** [comment]
@FNALbuild run build test

**Comment by kutschke** [comment]
@FNALbuild run build test

**Pull request merged**